### DISCUSSION TITLE
pymodbus

### DISCUSSION THREAD

**Discussion by Ricdo7** [pymodbus-dev/pymodbus#1412] [Q&A] [answered]
Greeting,
I don't know how to establish communication between python and machine expert builder. I want to start by establishing only communication on the simulated PLC. After that, create an algorithm in python that I will simulate in machine expert builder. In python, I need to put the IP address of the PLC, which for some reason does not show me in the simulated mode of operation in machine exper.
Can someone help me?
![Screenshot_2](https://user-images.githubusercontent.com/127210399/223517812-a757d76a-7f07-440b-a8bb-63f1626db757.png)

**Comment by janiversen** [comment, marked as answer]
Read the documentation and view the examples, that is the help we can offer.
